lib/features/home_view/home_view.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:catchqr/features/home_view/widgets/recetly_active_section.dart';
+import 'package:flutter/material.dart';
+
+class HomeView extends StatelessWidget {
+  const HomeView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return  Scaffold(
+      appBar: AppBar(
+        title: const Text('Home'),
```

Comment:
I suppose this is a placeholder widget. Cause if it is, then cool (maybe some comment). If not, then this should be a localized string :)

Reply:
yea it's placeholder I'll add comment